### PR TITLE
When outputing to terraform, don't say the cluster is starting

### LINKED ISSUE #1141: kops says it is starting the cluster when it isn't
I ran:

```
 kops create cluster ${CLUSTER_NAME} \
    --cloud aws \
    --target terraform \
    $SOME_OTHER_FLAGS
```

And sometime later I ran ``` kops update cluster ${CLUSTER_NAME} --target=terraform```

The update erroneously (reports that it is starting the cluster and creating resources)
```
kops update cluster $CLUSTER_NAME --target=terraform
I1212 13:51:20.307311    6131 executor.go:68] Tasks: 0 done / 61 total; 29 can run
I1212 13:51:20.310590    6131 dnszone.go:156] Check for existing route53 zone to re-use with name "example.com"
I1212 13:51:20.403505    6131 dnszone.go:163] Existing zone "example.com." found; will configure TF to reuse
I1212 13:51:20.881620    6131 vfs_castore.go:384] Issuing new certificate: "master"
I1212 13:51:21.304439    6131 vfs_castore.go:384] Issuing new certificate: "kubecfg"
I1212 13:51:21.554309    6131 vfs_castore.go:384] Issuing new certificate: "kubelet"
I1212 13:51:22.341021    6131 executor.go:68] Tasks: 29 done / 61 total; 11 can run
I1212 13:51:22.426379    6131 executor.go:68] Tasks: 40 done / 61 total; 17 can run
I1212 13:51:22.678702    6131 executor.go:68] Tasks: 57 done / 61 total; 4 can run
I1212 13:51:22.679638    6131 executor.go:68] Tasks: 61 done / 61 total; 0 can run
I1212 13:51:22.686118    6131 target.go:165] Terraform output is in out/terraform
I1212 13:51:22.888553    6131 update_cluster.go:150] Exporting kubecfg for cluster
Wrote config for example.com to "/Users/mrwacky42/.kube/config"

Cluster is starting.  It should be ready in a few minutes.

Suggestions:
 * list nodes: kubectl get nodes --show-labels
 * ssh to the master: ssh -i ~/.ssh/id_rsa admin@api.example.com
 * read about installing addons: https://github.com/kubernetes/kops/blob/master/docs/addons.md
```

If I run `kops update cluster ...` again (still having not executed the Terraform code), it does not report that the cluster is starting:
```
kops update cluster ${CLUSTER_NAME} --target=terraform
I1212 14:12:19.899843    6563 executor.go:68] Tasks: 0 done / 61 total; 29 can run
I1212 14:12:19.901893    6563 dnszone.go:156] Check for existing route53 zone to re-use with name "example.com"
I1212 14:12:19.990657    6563 dnszone.go:163] Existing zone "example.com." found; will configure TF to reuse
I1212 14:12:23.004469    6563 executor.go:68] Tasks: 29 done / 61 total; 11 can run
I1212 14:12:23.090742    6563 executor.go:68] Tasks: 40 done / 61 total; 17 can run
I1212 14:12:23.351288    6563 executor.go:68] Tasks: 57 done / 61 total; 4 can run
I1212 14:12:23.351788    6563 executor.go:68] Tasks: 61 done / 61 total; 0 can run
I1212 14:12:23.360307    6563 target.go:165] Terraform output is in .
I1212 14:12:23.503906    6563 update_cluster.go:150] Exporting kubecfg for cluster
Wrote config for example.com to "/Users/mrwacky42/.kube/config"
```

### PR DESCRIPTION
**Pull request by justinsb**
Suggest the user goes into the output directory and runs terraform plan
& apply instead.

Fix #1141

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1382)

### CODE REVIEW

File: cmd/kops/update_cluster.go [hunk 1]
```diff
@@ -203,9 +203,16 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 
 		if !hasKubecfg {
 			// Assume initial creation
-			fmt.Printf("\n")
-			fmt.Printf("Cluster is starting.  It should be ready in a few minutes.\n")
-			fmt.Printf("\n")
+			if c.Target == cloudup.TargetTerraform {
+				fmt.Printf("\n")
+				fmt.Printf("Terraform output has been placed into %s\n", c.OutDir)
+				fmt.Printf("You likely want to run terraform plan and apply in that directory\n")
```
Comment (chrislovecnm) [inline cmd/kops/update_cluster.go]: Should we show the commands?  Might be helpful.

File: cmd/kops/update_cluster.go [hunk 2]
```diff
@@ -203,9 +203,16 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
...
+				fmt.Printf("\n")
+			} else {
+				fmt.Printf("\n")
+				fmt.Printf("Cluster is starting.  It should be ready in a few minutes.\n")
```
Comment (chrislovecnm) [inline cmd/kops/update_cluster.go]: > Cluster is starting.  It should be ready in a few minutes.

Not a fan telling our users "should".  You want me to file an issue to improve how this reads?

Comment (justinsb) [inline cmd/kops/update_cluster.go]: No need for an issue - just go straight to PR!

File: cmd/kops/update_cluster.go (hunk 1 above)
Comment (justinsb) [inline cmd/kops/update_cluster.go]: Added commands

**Pull request merged**